vsg/__main__.py

```diff
@@ -42,6 +43,42 @@ def parse_command_line_arguments():
         return parser.parse_args()
 
 
+def get_installed_styles():
```

Comment:
`get_built_in_styles` / `get_builtin_styles` ?

Reply:
That makes sense.

Reply:
In case of change to predefined, it would be `get_predefined_styles`.